tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -46,4 +50,39 @@ func Test_E2E_Provisioning(t *testing.T) {
 
 	err = ts.dashboardChecker.AssertRedirectedToUAA(dashboardURL)
 	assert.NoError(t, err)
+
+	if ts.IsTestAzureEventHubsEnabled {
+		checkAzureEventHubProperties(ts, t, operationID)
+	}
+}
+
+func checkAzureEventHubProperties(ts *Suite, t *testing.T, operationID string) {
+	filter := fmt.Sprintf("tagName eq 'InstanceID' and tagValue eq '%s'", ts.InstanceID)
+	groupListResultPage, err := (*ts.azureClient).ListResourceGroup(context.TODO(), filter, nil)
+
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, groupListResultPage.Response().StatusCode, "HTTP GET fails for ListResourceGroup")
+	assert.Equal(t, 1, len(groupListResultPage.Values()), "groupListResultPage should return 1 ResourceGroup")
+
+	rg := groupListResultPage.Values()[0]
+	tagsRG := groupListResultPage.Values()[0].Tags
+
+	// Check tags for ResourceGroup
+	assert.Equal(t, operationID, *tagsRG[azure.TagOperationID], "Tag OperationID for ResourceGroup is incorrect")
```

Comment:
Edge case: what if the key is not inside the map. Then nil will be dereferenced => can we add a check for nil ? :) Same for other map tag keys

Reply:
checking for "nil"s now ;P